### DISCUSSION TITLE
Funding from NLnet

### DISCUSSION THREAD

**Discussion by jakoschiko** [Rust-GPU/rust-gpu#3] [General]
[NLnet](https://nlnet.nl/) is funding [a broad set of open technologies](https://nlnet.nl/project/current.html), e.g. [GPGPU Playground](https://nlnet.nl/project/GPGPU/) and [Renderling](https://nlnet.nl/project/Renderling/), and is accepting [new applications](https://nlnet.nl/funding.html). I made good experiences with them. They seem to like Rust projects and have low bureaucratic overhead. Did you already consider to ask them for funding? I'd be happy if this project would thrive.

**Comment by LegNeato** [comment]
Some maintainers are already getting this funding as pointed out. Thank you!
